src/components/Patient/PatientDetailsTab/Demography.tsx

```
@@ -0,0 +1,454 @@
+          name={
+            patientData.assigned_to_object
+              ? formatName(patientData.assigned_to_object)
+              : undefined
+          }
+        />,
+      ],
+    },
+    {
+      id: "insurance-details",
+      details: [
+        <div className="w-full md:col-span-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)